torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2056,6 +2057,18 @@ Tensor tensorexpr::computeOperandValue(
           kIntegralTypes | kFloatingPointTypes | kBoolType);
     } break;
 
+    case aten::sign: {
+      return computeOneOperand(
+          "aten_sign",
+          inputs,
+          outputShape,
+          outputType,
+          [](const ExprHandle& a) {
+            return CompareSelect::make(
+                a, ExprHandle(0), ExprHandle(-1), ExprHandle(1), kLT);
```

Comment:
I was expecting that this would be incorrect for -0.0 but...
```
>>> torch.tensor([-0.0, 0.0]).sign()
tensor([0., 0.])
```

Seems fishy.

Reply:
Oh wait, I see what's going on.  torch.sign returns 0 for +/-0.0: https://pytorch.org/docs/stable/generated/torch.sign.html.  So this expression is wrong for either flavor of 0 :-)

Reply:
Thanks for pointing those out. Handled all those cases now.